conf/tz/tz-src-url.txt

```diff
@@ -1 +1 @@
-ftp://ftp.iana.org/tz/releases/tzdata2019b.tar.gz
+ftp://ftp.iana.org/tz/releases/tzdata2021a.tar.gz
```

Comment:
there are some other files also needs to be updated, check last update https://github.com/Zimbra/zm-timezones/pull/16/files

Reply:
Tools `tools/updateWindowsNames.py` & `tools/checkPrimaryZones.py` don't give any changes for the windows-names or extra-data as in the above PR. 
Tool `tools/TzMsgCheck.pl -t ../../timezones.ics` gives o/p of removal lots entries like below, looking into it.

`TIMEZONE entry Africa/Algiers should be removed from ../../../messages/TzMsg.properties`
`TIMEZONE entry Africa/Cairo should be removed from ../../../messages/TzMsg.properties`
`TIMEZONE entry Africa/Casablanca should be removed from ../../../messages/TzMsg.properties`
`TIMEZONE entry Africa/Harare should be removed from ../../../messages/TzMsg.properties`
`TIMEZONE entry Africa/Khartoum should be removed from ../../../messages/TzMsg.properties`
`TIMEZONE entry Africa/Monrovia should be removed from ../../../messages/TzMsg.properties`
`TIMEZONE entry Africa/Nairobi should be removed from ../../../messages/TzMsg.properties`
`TIMEZONE entry Africa/Sao_Tome should be removed from ../../../messages/TzMsg.properties`
`TIMEZONE entry Africa/Tripoli should be removed from ../../../messages/TzMsg.properties`
`TIMEZONE entry Africa/Windhoek should be removed from ../../../messages/TzMsg.properties`
`TIMEZONE entry America/Adak should be removed from ../../../messages/TzMsg.properties`
`TIMEZONE entry America/Anchorage should be removed from ../../../messages/TzMsg.properties`

Reply:
Looks like there was some issue in this ics file generation, I tried and got updates in extra-data and windows-name files, check changes in https://github.com/Zimbra/zm-timezones/compare/saket?expand=1
Although I am not able to fully generate timezones.ics file and getting below error
```
zimbra@apps-development:/tmp/zm-timezones/conf/tz$ ./zmtzdata2ical -o timezones.ics --old-timezones-file ../timezones.ics -e extra-data -t tzdata windows-names
Problem loading old timezones.ics - ignoring it.  An error ocurred during parsing - line: 3439
```